Actions/Deliver/Deliver.ps1

```diff
@@ -199,7 +198,7 @@ try {
             #Calculate the folders per artifact type
             'Apps', 'TestApps', 'Dependencies' | ForEach-Object {
                 $artifactType = $_
-                $singleArtifactFilter = "*-$refname-$artifactType-*.*.*.*";
+                $singleArtifactFilter = "$project-$refname-$artifactType-*.*.*.*";
```

Comment:
Using `$project` might not work as it is supposed to be a path, right? Like `root/folder/subfolder/projectFolder`.

Maybe use `$projectName`? (From above: `$projectName = $project -replace "[^a-z0-9]", "-"`).

However, when we calculate artifacts names in CICD, the logic is slightly different: `$($project.Replace('\','_'))` (see [here](https://github.com/microsoft/AL-Go-PTE/blob/main/.github/workflows/CICD.yaml#:~:text=%24(%24project.Replace(%27%5C%27%2C%27_%27)))).

Reply:
Good catch.
$project should be the name of the published artifact (/ and \ converted to _).
In CI/CD it will today only have a \ but soon - they might have / as well - have added the same conversion all over.
Should refactor and create a function for this, but in many of the areas, no function library is available.

projectName is for blob storage and other things where all special characters are stripped off - for output purposes.